README.md

```diff
@@ -38,7 +38,7 @@ npm start
 - [Polymer](https://github.com/webcomponents/custom-elements-everywhere/tree/master/libraries/polymer)
 - [Preact](https://github.com/webcomponents/custom-elements-everywhere/tree/master/libraries/preact)
 - [React](https://github.com/webcomponents/custom-elements-everywhere/tree/master/libraries/react)
-- [React@experimental](https://github.com/webcomponents/custom-elements-everywhere/tree/master/libraries/react-experimental)
+- [React 19](https://github.com/webcomponents/custom-elements-everywhere/tree/master/libraries/react-19)
```

Comment:
Here and elsewhere, let's just call it "React Beta", since once it's released stable  it'll still be called "React 19" but here we'll want to just call it "React"